node/src/BaseClient.ts

```diff
@@ -482,7 +482,7 @@ export type BaseClientConfiguration = {
      */
     readFrom?: ReadFrom;
     /**
-     * Choose the Redis protocol to be used with the server.
+     * Choose the Valkey protocol to be used with the server.
```

Comment:
```suggestion
     * Choose the protocol to be used with the server.
```
The protocol name is still REdis Serialization Protocol

Reply:
I would argue that Valkey's protocol IS just RESP. But this is probably less confusing.